tests/upgrade/integration/v2/generated/post/query - module-version-map/v2.json

```diff
@@ -23,11 +23,11 @@
             },
```

Comment:
I've highlighted the relevant places whence the `consensus` module is missing within #690.

Let's add it here as well, as it wasn't registered at all, with the test giving false positives.

Reply:
Its version should be `1`.